.emacs.d/config.org

```diff
@@ -530,6 +530,15 @@ blacklists a few modes, it belongs in Global.
 
 #+END_SRC
 
+** Terraform
+
+#+BEGIN_SRC emacs-lisp
+
+  (use-package terraform-mode
+    :ensure t)
```

Comment:
I know you're copying and pasting other things I put in here, but the extra `ensure` isn't necessary because I set `(setq use-package-always-ensure t)` when setting up use-package.

Reply:
gtk